src/messages/MOSDOp.h

```diff
@@ -36,7 +36,7 @@ namespace _mosdop {
 template<typename V>
 class MOSDOp final : public MOSDFastDispatchOp {
 private:
-  static constexpr int HEAD_VERSION = 9;
+  static constexpr int HEAD_VERSION = 8;
```

Comment:
This commit creates as short-living version `9` which resembles a similar situation in RocksDB. This seems fine to me as it's all about `main` but I want to explicit here: we provide NO GUARANTEES on dencoding interoperability between two **pure**-`main` revision, right?

CC: @gregsfortytwo, @athanatos.

Reply:
@rzarzynski I think the main issue is unblocking failing tests where we have older versions of OSDs

Reply:
Right @rzarzynski, no guarantees on versioning in main. I mean, we don’t love it, but we can’t guarantee it for precisely situations like this, where we discover an issue in integration tests.